vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -103,6 +103,7 @@
     </Link>
     <Midl>
       <AdditionalIncludeDirectories>$(ReactNativeWindowsDir)\Microsoft.ReactNative;$(ReactNativeWindowsDir)\Microsoft.ReactNative.Cxx;</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>CORE_ABI</PreprocessorDefinitions>
```

Comment:
It's best if we accumulate the preprocessor definitions, in case others have already been defined.
Please change to:
```
<PreprocessorDefinitions>CORE_ABI;%(PreprocessorDefinitions)</PreprocessorDefinitions>

Reply:
Fixed in iteration 2.